web/controllers/userController/format.ts

```diff
@@ -192,6 +192,7 @@ export class ApiGetUser {
     registrationDate: number;
     expertData?: ApiExpertData;
     pupilTutoringInterestConfirmationStatus?: InterestConfirmationStatus;
+    isOfficial?: boolean;
```

Comment:
What exactly is "official" supposed to mean? Could you clarify this, best by adding a comment to this attribute.

Reply:
`isOfficial` is actually already used in several other parts of the code. It essentially means that the student is an "intern", i.e. they get credit for their work for us at their university. But yes, a comment explaining this somewhere wouldn't do any harm.